server/model/comment.js

```diff
@@ -0,0 +1,31 @@
+function comment() {
+    this.id = -1
+    this.parent_id = -1
+    this.parent_type = ""
+    this.comment_text = ""
+    this.user_id = -1
+    this.created_at = new Date()
+    this.updated_at = new Date()
```

Comment:
We should add terminating semicolons here. Although filled in by JavaScript's ASI, current convention across the other files is to include them, so we should do so to maximize readability and consistency across the codebase.